cmd/di.go

```diff
@@ -465,11 +466,39 @@ func (di *Dependencies) bootstrapIdentityComponents(options node.Options) {
 	di.IdentityRegistration = identity_registry.NewRegistrationDataProvider(di.Keystore)
 }
 
-func (di *Dependencies) bootstrapLocationComponents(options node.OptionsLocation, configDirectory string) (err error) {
+func (di *Dependencies) bootstrapLocationComponents(options node.OptionsLocation, configDirectory string) error {
 	di.IPResolver = ip.NewResolver(options.IPDetectorURL)
-	di.LocationResolver, err = location.CreateLocationResolver(di.IPResolver, options.Country, options.City, options.Type, options.NodeType, options.Address, options.ExternalDb, configDirFlag)
-	if err != nil {
-		return err
+
+	manualResolver := location.NewStaticResolver(options.Country, options.City, options.NodeType, di.IPResolver)
+
+	switch options.Type {
+	case node.LocationTypeManual:
+		di.LocationResolver = location.NewFallbackResolver([]location.Resolver{manualResolver})
+
+	case node.LocationTypeBuiltin:
+		var builtinResolver location.Resolver
+		builtinResolver, err := location.NewBuiltInResolver(di.IPResolver)
+		if err != nil {
+			log.Error("Failed to load builtin location resolver: ", err)
+			builtinResolver = location.NewFailingResolver(err)
+		}
+		di.LocationResolver = location.NewFallbackResolver([]location.Resolver{builtinResolver, manualResolver})
```

Comment:
Is it a good idea to add `manualResolver` everywhere like the last one? 
Most likely it will contain empty values if user set different type of resolver.
Maybe it's better to return an error then empty values?

Reply:
As I user I should be able to start a node if location auto-detection is not working.
By entering location attributes manually

Reply:
But this will result in proposals being announced with no country, as to use the manual resolver - you'll have to provide the flags in order for it to have any values. Is it not better to fail then, and inform the user that he can override the location resolution via flags rather than defaulting to empty values? I'd rather fallback to the builtin DB...

Reply:
Reverted to have only single type resolving currently, no falbacking until:
- missing city/ASN resolving in `buildin` resolver
- no error handing for `manual` resolver